session/src/main/scala/akka/cassandra/session/javadsl/CassandraSession.scala

```diff
@@ -71,14 +71,24 @@ final class CassandraSession(delegate: akka.cassandra.session.scaladsl.Cassandra
   def underlying(): CompletionStage[Session] =
     delegate.underlying().toJava
 
+  /**
+   * Execute <a href=https://docs.datastax.com/en/archived/cql/3.3/cql/cql_reference/cqlCommandsTOC.html">CQL commands</a>
+   * to manage database resources (create, replace, alter, and drop tables, indexes, user-defined types, etc).
+   *
+   * The returned `CompletionStage` is completed when the command is done, or if the statement fails.
+   */
+  def executeDdl(stmt: String): CompletionStage[Done] =
+    delegate.executeDdl(stmt).toJava
+
   /**
    * See <a href="http://docs.datastax.com/en/cql/3.3/cql/cql_using/useCreateTableTOC.html">Creating a table</a>.
    *
    * The returned `CompletionStage` is completed when the table has been created,
    * or if the statement fails.
    */
+  @deprecated("Use executeDdl instead.", "0.xx")
```

Comment:
I'm not sure which version to use here. This is why the PR is a draft for now.

Reply:
since 0.99 has been released this will be included in 0.100 (and upcoming 1.0.0)

Reply:
Thanks, @patriknw. I was in doubt between `0.100` or `1.0.0`.